tests/test_tools.py

```diff
@@ -218,13 +218,22 @@ def test_subdomains(self):
         assert openreview.tools.subdomains('   ') == []
 
     def test_replace_members_with_ids(self, client, test_client):
+        new_group = client.post_group(
```

Comment:
no need to create the group, once a member is added to a group the backend created the group automatically.